java/src/jmri/jmrit/logixng/implementation/swing/AbstractMaleSocketSwing.java

```diff
@@ -128,7 +128,10 @@ public final void updateObject(@Nonnull Base object) {
         if (! (object instanceof AbstractMaleSocket)) {
             throw new IllegalArgumentException("object is not an AbstractMaleSocket: " + object.getClass().getName());
         }
-        
+        if ( errorHandlingComboBox == null || catchAbortExecutionCheckBox == null ){
```

Comment:
I think the best way in this case is to change lines 25-26:
```
private final JComboBox<ErrorHandlingType> errorHandlingComboBox = new JComboBox<>();
private final JCheckBox catchAbortExecutionCheckBox = new JCheckBox();
```
They can both be final.